packages/sap-systems-ext-webapp/.npmignore

```diff
@@ -0,0 +1,8 @@
+test/
+config/
+reports/
+esbuild.js
+tsconfig*.json
+
+*.tsbuildinfo
+**/*.tsbuildinfo
```

Comment:
Probably cleaner to remove this file and use `files` property in package.json to specify files to include in the package tgz.